openid-federation-1_0.xml

```diff
@@ -6158,7 +6180,9 @@ redirect_uri=https%3A%2F%2Frp.example.com%2Fauthz_cb
 	    If the OP fails to establish trust with the RP,
```

Comment:
Please add: 

or finds the RP metadata to be invalid or conflicting metadata policy